src/test/cpp/gtester/ParsePopulationModel.cpp

```diff
@@ -10,7 +10,7 @@ namespace Tests {
 
 TEST(ParsePopulationModel, ParseDefaultPopulationModel)
 {
-	std::ifstream pop_file{"data/population_model_default.xml"};
+	std::ifstream pop_file{"../data/population_model_default.xml"};
```

Comment:
Prepending `../` to the path kind of bothered me as well.

[This file](https://github.com/flu-plus-plus/bachelorproef/blob/master/src/test/cpp/gtester/CMakeLists.txt) seems to set the working directory for the tests. We might be able to set that to a different value. I'm worried that might break the existing stride tests though, because those load files as well.